.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@ This is a (multiple allowed):
 * [ ] feature-discussion (RFC)
 
 ## DESCRIPTIVE TITLE HERE
```

Comment:
Let's start with keeping it at bottom and if we still keep getting an annoying amount of question we can consider moving it at the top.